blog/2025/pcsx2-2.6/index.mdx

```diff
@@ -0,0 +1,284 @@
+  ### Organize Game Screenshot by Name in Folders
+</PCSX2PRLink>
+
+Have you ever been in a situation where your screenshot folder turn into a disorganized mess of game capture and snapshots? especially when you use the feature a lot.
+
+Thanks to fthomys, You can now organize screenshot folder by its game's name.
+
+<Image src={require("./img/screenshot_folder.webp").default} />
+
+<PCSX2PRLink authors="SternXD" prNums="13590">
+  ### Add Indicators for Texture Dumping/Replacement
+</PCSX2PRLink>
+
+Ever wondered if your texture replacement is working? Tried eye-balling it to no avail? Worry no more! You can now enable the Texture Dumping/Replacement indicator to find out if you have the textures being dumped/replaced correctly.
+
+<Image src={require("./img/texrep.webp").default} />
+
+## Performance Improvements
+
+<PCSX2PRLink
+  authors="TheLastRar,lightningterror"
+  prNums="13612,13575,13539,13549,13653,13339"
+>
+  ### Various Blending and Hardware Renderer Improvements
+</PCSX2PRLink>
+
+#### Feedback Reads on Direct3D 12 ([#13653](https://github.com/PCSX2/pcsx2/pull/13653))
+
+Upon looking though the Direct3D 12 documentation, it was found that aliased resources (2 resources sharing the same memory) could be used to bind the same texture as both a shader resource and render target.
+This relies on the two resources being able to inherit data from each other, the requirements of which can be satisfied using simultaneous access textures.
+
+This allows us to speed the Direct3D 12 renderer up substantially, with some systems seeing over 5x improvements on specific games.
+
+<BarChart chartDataUrl={require("./charts/Chart-gameperf-pt2.yaml").default} />
+
+While NVIDIA and Arc had no issues, lightningterror discovered that some AMD GPUs showed issues. While the Direct3D 12 documentation implies that data inheritance should work even with undefined memory layouts, the texture layout needed to be specified for some generations of AMD GPUs. Once this issue was identified it was fortunately an easy fix to implement.
+
+While reviewing the code, lightningterror spotted and fixed a regression with handling stencil date one.
+
+There has also been other various enhancement to the hardware renderer where various games such as Enthusia gained a significant 9% improvements in Direct3D 11.
+
+<BarChart chartDataUrl={require("./charts/Chart-gameperf-pt1.yaml").default} />
+
+<PCSX2PRLink authors="TheLastRar,govanify" prNums="13682,13697,13614">
+  ### The Resizing Saga
+</PCSX2PRLink>
+
+It all started with [#13613](https://github.com/PCSX2/pcsx2/issues/13613), one of our tester, JordanTheToast who recently upgraded to an Radeon RX 9060 XT, noticed that resizing PCSX2 while in-game causes massive lag/delay during the resize. This ended up turning into a massive leap of faith towards a deep rabbit hole that confuses the team for days. A combination of GPU driver bugs to the way Qt handle window resizing slowly uncovers the problem.
```

Comment:
Consider breaking this up a bit. Feels a bit run-on, a few minor grammar errors. The leap of faith reference doesn't quite fit the scenario imo.

"It all started with [#13613](https://github.com/PCSX2/pcsx2/issues/13613). One of our testers, JordanTheToast had recently upgraded to a Radeon RX 9060 XT. He noticed that resizing PCSX2 while in-game causes massive lag/delay during the resize. This ended up turning into a deep dive that confused the team for days. A look into GPU driver bugs, and the way Qt handles window resize events slowly uncovered the problem."